app/src/main/java/com/x8bit/bitwarden/ui/platform/components/dialog/BitwardenPINDialog.kt

```diff
@@ -0,0 +1,78 @@
+package com.x8bit.bitwarden.ui.platform.components.dialog
+
+import androidx.compose.foundation.layout.imePadding
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.semantics.testTagsAsResourceId
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.components.button.BitwardenTextButton
+import com.x8bit.bitwarden.ui.platform.components.field.BitwardenPasswordField
+
+/**
+ * Represents a Bitwarden-styled dialog for the user to enter their PIN.
+ *
+ * @param onConfirmClick called when the confirm button is clicked and emits the entered PIN.
+ * @param onDismissRequest called when the user attempts to dismiss the dialog (for example by
+ * tapping outside of it).
+ */
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun BitwardenPINDialog(
```

Comment:
⛏️ Should be `Pin` instead of `PIN`. There are a couple other instances of `PIN` in other classes that need to be corrected also.